customerOrders/README.md

```diff
@@ -0,0 +1,57 @@
+## Generate the persist client object, types, and scripts
+
+1. Run `$ bal persist generate`.
+
+2. Edit the generated `Config.toml` file to add the database configurations.
+
+```
+    [customerOrders]
+    host = "localhost"
+    port = 3305
+    user = "root"
+    password = "sqletl123"
+    database = "customer_order"
+```
+The `password` should be replaced with the actual password that will be used for `MYSQL_ROOT_PASSWORD` when setting up the database.
+
+## Setting up the MySQL database on docker
+
+1. Pull the official docker image for MySQL and run it on a docker container
+
+```sh
+    $ docker run --rm -d --name mysql-etl -p 3305:3306 -e MYSQL_ROOT_PASSWORD=sqletl123 mysql:latest
+```
```

Comment:
Shall we not use docker directly but a docker-compose, so that we can mount and run the init script in it directly?

Reply:
You can remove steps 2-6 then

Reply:
docker-compose.yml added to run steps from line 22 to 54